data/blog/session-replay-sdk-bundle-size-optimizations.md

```diff
@@ -0,0 +1,208 @@
+---
+title: 'Session Replay SDK Bundle Size Optimizations'
+date: '2023-10-31'
+tags: ['sdk', 'javascript', 'replay', 'session-replay']
+draft: false
+summary: 'An in-depth analysis of how we managed to cut the Session Replay SDK bundle size by X%.'
+images: [/images/session-replay-sdk-bundle-size-optimizations/hero.jpg]
+layout: PostLayout
+canonicalUrl: https://sentry.engineering/blog/session-replay-sdk-bundle-size-optimizations
+authors: ['francesconovy']
+---
+
+[Bundle Size matters](https://blog.sentry.io/js-browser-sdk-bundle-size-matters/) - this is something we SDK engineers at Sentry are acutely aware of.
```

Comment:
```suggestion
[Bundle Size matters](https://blog.sentry.io/js-browser-sdk-bundle-size-matters/) - this is something we SDK engineers at Sentry are actually aware of.
```

Reply:
I think acutely is more accurate, but maybe we can find a more straightforward term for this 🤔 maybe:

...always are of?

or something like this?

Reply:
oh that's right, somehow I read a typo there instead of taking the word in.